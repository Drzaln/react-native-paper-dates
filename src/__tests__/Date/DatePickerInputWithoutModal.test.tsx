import { render } from '@testing-library/react-native'
import DatePickerInputWithoutModal from '../../Date/DatePickerInputWithoutModal'

it('renders DatePickerInputWithoutModal', () => {
  const { toJSON } = render(
    <DatePickerInputWithoutModal
      locale="en"
      value={new Date()}
      onChange={() => null}
      inputMode="start"
    />
  )
  expect(toJSON()).toMatchSnapshot()
})
