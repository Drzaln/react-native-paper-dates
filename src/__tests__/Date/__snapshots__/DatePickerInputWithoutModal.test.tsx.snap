// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders DatePickerInput 1`] = `null`;
